### PR TITLE
73387 Upload + 73388 Download attachment from tag

### PR DESCRIPTION
**Pull request by janid1967**
Both Upload and Download endpoints for TagAttachment

### CODE REVIEW

File: src/tests/Equinor.Procosys.Preservation.Query.Tests/GetTagAttachment/GetTagAttachmentQueryHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            var options = new AttachmentOptions
+            {
+                BlobContainer = BlobContainer
+            };
+
+            _attachmentOptionsMock
+                .Setup(x => x.CurrentValue)
+                .Returns(options);
+
+            using var context = new PreservationContext(dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider);
```
Comment (Cnordbo) [inline src/tests/Equinor.Procosys.Preservation.Query.Tests/GetTagAttachment/GetTagAttachmentQueryHandlerTests.cs:44]: Are we missing brackets on the using statement here?

File: src/tests/Equinor.Procosys.Preservation.Query.Tests/GetTagAttachment/GetTagAttachmentQueryHandlerTests.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+            _testDataSet = AddTestDataSet(context);
+
+            var tag = _testDataSet.Project1.Tags.First();
+
+            _attachment = new TagAttachment(TestPlant, new Guid("{C3412890-1EF8-4E34-B96C-5488200A5AF5}"), "FileA");
```
Comment (Cnordbo) [inline src/tests/Equinor.Procosys.Preservation.Query.Tests/GetTagAttachment/GetTagAttachmentQueryHandlerTests.cs:50]: It doesnt seem like this value is represented elsewhere, we could just use 
```c#
Guid.NewGuid()
```
Brackets are not needed to create the GUID.

File: src/tests/Equinor.Procosys.Preservation.Query.Tests/GetTagAttachments/GetTagAttachmentsQueryHandlerTests.cs [hunk 3]
```diff
@@ -26,7 +26,7 @@ protected override void SetupNewDatabase(DbContextOptions<PreservationContext> d
 
                 var tag = _testDataSet.Project1.Tags.First();
 
-                _attachment = new TagAttachment(TestPlant, new Guid("{C3412890-1EF8-4E34-B96C-5488200A5AF5}"), "TitleA", "FileA");
+                _attachment = new TagAttachment(TestPlant, new Guid("{C3412890-1EF8-4E34-B96C-5488200A5AF5}"), "FileA");
```
Comment (Cnordbo) [inline src/tests/Equinor.Procosys.Preservation.Query.Tests/GetTagAttachments/GetTagAttachmentsQueryHandlerTests.cs:29]: It doesnt seem like this value is represented elsewhere, we could just use 
```c#
Guid.NewGuid()
```
Brackets are not needed to create the GUID.

File: src/tests/Equinor.Procosys.Preservation.Query.Tests/GetTagAttachment/GetTagAttachmentQueryHandlerTests.cs (hunk 1 above)
Comment (janid1967) [reply src/tests/Equinor.Procosys.Preservation.Query.Tests/GetTagAttachment/GetTagAttachmentQueryHandlerTests.cs:44]: No. C# 8 (or 7)

**Pull request merged**